docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Sharing To & From GitHub Gists
+description: GitHub Gists are a great way to share code with a group of people for public visibility, and integrate your snippets with your GitHub repositories. Pieces makes it easy to update your Gist repository by enabling you to share saved code snippets within Pieces as Gists, right from the desktop app.
+---
+
+# Import Gists from GitHub
+
+<Image zoom src="https://storage.googleapis.com/pieces_static_resources/pfd_wiki/IMPORT_GIST.gif" />
+
+Before proceeding to work with Gists from GitHub in Pieces, make sure that you are signed in. _If you are signed in to both Google and GitHub, be sure that the **primary emails on both accounts match**._
+
+## Sharing Pieces snippets as GitHub Gists
+
+GitHub Gists are a great way to share code with a group of people for public visibility, and integrate your snippets with your GitHub repositories. Pieces makes it easy to update your Gist repository by enabling you to share saved code snippets within Pieces as Gists, right from the desktop app.
+
+<Image zoom alt={"Generating a shareable link in the Pieces desktop app"} src="https://storage.googleapis.com/pieces_static_resources/pfd_wiki/SHAREABLE_LINK.gif" />
+
+1. After signing in and ensuring that you are **connected to the cloud**, select any snippet in the desktop app.
```

Comment:
would link this sentence to another helpful part of the docs

Reply:
this also could be betters as "After signing in and ensuring..." and then starting the steps to share pieces as github gists